src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -151,7 +151,7 @@ private void registerMiscItems() {
             new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE)
         );
 
-        registerItem(Items.LAPOTRON_CRYSTAL, RecipeType.ENHANCED_CRAFTING_TABLE,
+        registerItem(SlimefunItems.POWER_CRYSTAL, RecipeType.ENHANCED_CRAFTING_TABLE,
```

Comment:
Now you are registering an already-registered item :NotLikeThis:
And the lapotron crystal is the second tier power crystal, not the normal one.

Reply:
i changed it shush check  359ad08e3167b602100d6ba056f800162c9d3df4